### ISSUE TITLE
Dependency problem with aiohttp

### ISSUE THREAD

**Issue by brandsimon** [enigmampc/catalyst#377]
Dear Catalyst Maintainers,

Before I tell you about my issue, let me describe my environment:

# Environment

* Operating System: `Linux DEV 4.17.2-1-ARCH #1 SMP PREEMPT Sat Jun 16 11:08:59 UTC 2018 x86_64 GNU/Linux`
* Python Version: `Python 3.6.5`
* Python Bitness: `64`
* How did you install Catalyst: `pip`
* Python packages: 
```
aiodns==1.1.1
aiohttp==3.3.2
alembic==0.9.9
asn1crypto==0.24.0
async-lru==0.1.0
async-timeout==3.0.0
attrdict==2.0.0
attrs==18.1.0
backcall==0.1.0
bcolz==1.2.1
boto3==1.7.45
botocore==1.10.45
Bottleneck==1.2.1
bumpversion==0.5.3
cchardet==2.1.1
ccxt==1.14.267
certifi==2018.4.16
cffi==1.11.5
chardet==3.0.4
click==6.7
coincurve==7.1.0
contextlib2==0.5.5
cryptography==2.2.2
cyordereddict==1.0.0
Cython==0.28.3
cytoolz==0.9.0.1
decorator==4.3.0
docutils==0.14
empyrical==0.2.1
enigma-catalyst==0.5.14
eth-abi==1.1.1
eth-account==0.2.3
eth-bloom==1.0.0
eth-hash==0.1.4
eth-keyfile==0.5.1
eth-keys==0.2.0b3
eth-pm==0.1.0a17
eth-rlp==0.1.2
eth-tester==0.1.0b26
eth-utils==1.0.3
hexbytes==0.1.0
idna==2.7
idna-ssl==1.0.1
intervaltree==2.1.0
ipython==6.4.0
ipython-genutils==0.2.0
jedi==0.12.0
jmespath==0.9.3
jsonschema==2.6.0
Logbook==1.4.0
lru-dict==1.1.6
lxml==4.2.2
Mako==1.0.7
MarkupSafe==1.0
multidict==4.3.1
multipledispatch==0.5.0
netdisco==1.5.0
netifaces==0.10.7
networkx==2.1
numexpr==2.6.5
numpy==1.14.5
pandas==0.19.2
pandas-datareader==0.6.0
parsimonious==0.8.0
parso==0.2.1
patsy==0.5.0
pexpect==4.6.0
pickleshare==0.7.4
pkginfo==1.4.2
plyvel==1.0.4
prompt-toolkit==1.0.15
ptyprocess==0.6.0
py-ecc==1.4.3
py-evm==0.2.0a18
py-solc==2.2.0
pycares==2.3.0
pycparser==2.18
pycryptodome==3.6.3
pyethash==0.1.27
Pygments==2.2.0
pysha3==1.0.2
python-dateutil==2.7.3
python-editor==1.0.3
pytz==2018.4
redo==1.6
requests==2.19.1
requests-file==1.4.3
requests-ftp==0.3.1
requests-toolbelt==0.8.0
rlp==1.0.1
s3transfer==0.1.13
scipy==1.1.0
semantic-version==2.6.0
simplegeneric==0.8.1
six==1.11.0
sortedcontainers==2.0.4
SQLAlchemy==1.2.8
statsmodels==0.9.0
tables==3.4.4
toolz==0.9.0
tqdm==4.23.4
traitlets==4.3.2
trie==1.3.7
twine==1.11.0
uPnPClient==0.0.8
urllib3==1.23
wcwidth==0.1.7
web3==4.4.0
websockets==5.0.1
wrapt==1.10.11
yarl==1.1.0
zeroconf==0.20.0
```

Now that you know a little about me, let me tell you about the issue I am
having:

I want to install enigma-catalyst==0.5.14 with pip.

* What did you expect to happen?
The installation is working fine, there are no conflics.

* What happened instead?
py-evm 0.2.0a18 has requirement aiohttp<3.0.0,>=2.3.1, but you'll have aiohttp 3.3.2 which is incompatible.

Here is how you can reproduce this issue on your machine:

## Reproduction Steps

1. `python3 -m venv .venv`
2. `.venv/bin/python -m pip install -U pip setuptools`
3. `.venv/bin/python -m pip install enigma-catalyst==0.5.14`
...

## What steps have you taken to resolve this already?

Installing web3 before installing enigma-catalyst
`.venv/bin/python -m pip install web3==4.3.0`

# Anything else?

web3 released a new version and set a fixed dependency for eth-pm, which causes the conflict.

Sincerely,
Simon Brand

**Comment by lenak25** [comment]
Thanks for reporting, we will investigate this.

**Comment by lucasgadams** [comment]
Hi i've had the same issue. Fresh environment created with venv.

Python 3.6.5
enigma-catalyst==0.5.14

```
pkg_resources.ContextualVersionConflict: (aiohttp 3.3.2 (/Users/lucasadams/Projects/sandbox/catalyst/env/lib/python3.6/site-packages), Requirement.parse('aiohttp<3.0.0,>=2.3.1'), {'py-evm'})
```

**Comment by lucasgadams** [comment]
Seems to be an issue with ccxt requiring a higher version of aiohttp

**Comment by lucasgadams** [comment]
Yea ccxt requires aiohttp>=3.0.1

**Comment by lenak25** [comment]
Hi, thanks for updating, we are working to fix issues with the pip installation. 
conda install should be o.k, so in the meantime, could you please use conda or try the following workaround using pip:
1. Creating a clean environment
2. Installing web3: `pip install web3==4.0.0b11`
3. Installing catalyst: `pip install enigma-catalyst`

**Comment by brandsimon** [comment]
This issue is  resolved with 0.5.15
